Documentation/gettingstarted/docker.rst

```diff
@@ -178,7 +179,7 @@ policy by running:
 ::
 
   $ cilium policy import l3_l4_policy.json
-  Revision: 1
+  Revision: 2
```

Comment:
Why 2? Is that the first policy you're loading?

Reply:
Yes, looks like right after starting cilium there's an empty policy loaded:
```sh
vagrant@vagrant:~/cilium/examples/getting-started$ cilium policy get
[]
Revision: 1
```

Reply:
Ah, I thought that counted as 0 or didn't count at all.